x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/formula/editor/formula_editor.tsx

```diff
@@ -651,7 +656,26 @@ export function FormulaEditor({
         'lnsIndexPatternDimensionEditor-isFullscreen': isFullscreen,
       })}
     >
-      <div className="lnsIndexPatternDimensionEditor__section lnsIndexPatternDimensionEditor__section--shaded">
+      {!isFullscreen && (
+        <EuiFormLabel
+          css={css`
+            margin-top: ${euiTheme.size.base};
+            margin-bottom: ${euiTheme.size.xs};
+          `}
+        >
+          {i18n.translate('xpack.lens.indexPattern.dimensionEditor.headingFormula', {
+            defaultMessage: 'Formula',
+          })}
+        </EuiFormLabel>
+      )}
+      <div
+        className="lnsIndexPatternDimensionEditor--shaded"
+        css={css`
+          border: ${!isFullscreen ? euiTheme.border.thin : 'none'};
+          border-radius: ${euiTheme.border.radius.medium};
```

Comment:
Should this be conditional on fullscreen as well?

```suggestion
          border-radius: ${!isFullscreen ? euiTheme.border.radius.medium : 0};
```

Reply:
Of course!